docs/Algorithm data sheets/kex_sidh_cln16.md

```diff
@@ -0,0 +1,60 @@
+Algorithm data sheet: `kex_sidh_cln16`
+=======================================
+
+Algorithm
+---------
+
+**Name:** SIDH-CLN16
+
+**Description:** Key exchange protocol proposed by Costello et al. [CLN16] based on the hidden supersingular isogeny problem. Instantiation of the SIDH scheme proposed by De Feo, Jao and Plut in [DJP11].
+
+**Supporting research:**
+
+- [CLN16] Craig Costello, Patrick Longa, and Michael Naehrig. Efficient algorithms for supersingular isogeny
+Diffie-Hellman. In *CRYPTO 2016*, volume 9814 of LNCS, pp 572-601. Springer, 2016. [https://eprint.iacr.org/2016/413](https://eprint.iacr.org/2016/413)
+- [DJP11] Luca De Feo and David Jao and Jérôme Plût. Towards quantum-resistant cryptosystems from supersingular elliptic curve isogenies. In *PQCRYPTO 2011*, volume 7071 of LNCS, pages 19-34. Springer, 2011. [https://eprint.iacr.org/2011/506](https://eprint.iacr.org/2011/506)
+
+
+Security
+--------
+
+**Security model:** Authenticated-links adversarial model
```

Comment:
I think that's fair, I included this model as it is proven in the DJP11 paper, but I missed a relevant conclusion from CLN16: `the Canneti-Krawczyk model assumes perfectly authenticated links which effectively forces adversaries to be passive eavesdroppers`. I'll update this. :)